simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java

```diff
@@ -45,6 +47,23 @@ StatusNotificationRequest createStatusNotification(int evseId, int connectorId,
         return payload;
     }
 
+    StatusNotificationRequest createStatusNotification(Evse evse, Connector connector, Instant currentTime) {
+        StatusNotificationRequest payload = new StatusNotificationRequest();
+        payload.setEvseId(evse.getId());
```

Comment:
why not chain these calls?

Reply:
This method was mainly copy-pasted. The difference is method signature. All other similar methods use setters. I would keep the same way for now. Besides there is logical-condition - chaining will not be a big gain.